spec.md

```diff
@@ -85,24 +85,28 @@ added to enable support for new features.
 These fields MUST be OPTIONAL and SHOULD be ignored by clients and servers
 that do not understand them.
 
-### Changes Since v2.11
-
-* Added `context` field to request body for provisioning a service instance (`PUT /v2/service_instances/:instance_id`) and updating a service instance (`PATCH /v2/service_instances/:instance_id`). Also added the [profile.md](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md) file describing RECOMMENDED usage patterns for environment-specific extensions and variations. `context` will replace the `organization_guid` and `space_guid` request fields in future versions of the specification. In the interim, both SHOULD be used to ensure interoperability with old and new implementations.
-* The specification now uses RFC2119 keywords to make it clearer when things are REQUIRED rather than OPTIONAL or RECOMMENDED.
-* Request and response parameters of type string, if present, MUST be a non-empty string.
-* Cleaned up text around status codes in responses, clarifying when they MUST be returned.
-* Added clarity around the `app_guid` request field.
-* Clarified the semantics of `plan_id` and `parameters` fields in the updating a service instance request body.
-* Clarified the default value of the `plan_updateable` field.
-* Clarified when `route_service_url` is REQUIRED and when brokers can return extra data in bindings.
+### Changes Since v2.12
```

Comment:
Again, links here to the feature would be very helpful.

Reply:
done